### PR TITLE
Fix can't play movies on Android

### PR DESCRIPTION
**Pull request by krmbn0576**
Some changes for playing movies on RMMV.

- Fix can't play movies on Android (videoElement.play() is required on touchend)
- Make sure that wait until movie ended
- on iPhone/iPod, play movie inline
- Delete two global variables and a global function

### CODE REVIEW

**Comment by krmbn0576** [comment]
Oh, I also changed *index.html* and add *js/libs/iphone-inline-video.browser.js*.
How do I change these files on this repository?

File: src/rpg_core/Graphics.js [hunk 1]
```diff
@@ -376,14 +371,7 @@ Graphics.playVideo = function(src) {
     this._video.onerror = this._onVideoError.bind(this);
     this._video.onended = this._onVideoEnd.bind(this);
     this._video.load();
-
-    if (Utils.isMobileSafari()) {
```
Comment (hiroppy) [inline src/rpg_core/Graphics.js:380]: Why did you delete this condition?

EDIT: 397, 991, and 1013 lines are also.

> on iOS8 & 9, play movis inline (for experience)

Is there no effect of change?

File: src/rpg_core/Graphics.js [hunk 2]
```diff
@@ -1009,13 +997,6 @@ Graphics._onVideoError = function() {
  */
 Graphics._onVideoEnd = function() {
     this._updateVisibility(false);
-
-    if (Utils.isMobileSafari()) {
```
Comment (hiroppy) [inline src/rpg_core/Graphics.js:1013]: same here.

File: src/rpg_core/Graphics.js (hunk 1 above)
Comment (krmbn0576) [reply src/rpg_core/Graphics.js:380]: I replaced them with this._videoLoading rather than deleting them,
because RMMV waited until the video ends in mobileSafari only.
I apologize that this PR is hard to understand because it has multiple changes.
From now on I will send a simple one in the future.

**Comment by ghost** [comment]
I will prepare index.html and libs directory.

**Comment by ghost** [comment]
I review this. great :heart:

**Comment by ghost** [comment]
Can you please close it and make a new PR ?
Project structure is dramatically changed.
